modules/haloRtdProvider.js

```diff
@@ -88,20 +142,20 @@ function getSegments(reqBidsConfigObj, onDone, config, userConsent) {
  * @param {Object} userConsent
  * @param {Object} userIds
  */
-function getSegmentsAsync(adUnits, onDone, config, userConsent, userIds) {
-  let reqParams = {}
+export function getSegmentsAsync(adUnits, onDone, config, userConsent, userIds) {
+  let reqParams = {};
   if (typeof config == 'object' && config != null && Object.keys(config).length > 0) {
-    reqParams = config.params
+    reqParams = config.params.requestParams;
   }
 
-  const url = `https://seg.halo.ad.gt/api/v1/rtb_segments`;
+  const url = `http://127.0.0.1:5000/api/v1/rtb_segments`;
```

Comment:
did you mean to commit this? looks like a dev request. fyi, I am not seeing segment responses using from the **seg.halo.ad.gt** endpoint using the example page and it's params.